src/Cli/OrgCommands.php

```diff
@@ -219,6 +222,10 @@ public function orgUpdateProjectsInfo($csv_file, $options = [
                     // If empty or invalid support level, we won't update it here.
                     if ($projectUpdateSupportLevel && (empty($projectSupportLevel) || !$this->validateProjectSupportLevel($projectSupportLevel))) {
                         $projectUpdateSupportLevel = false;
+                        if ($options['skip-on-empty-support-level']) {
+                            $this->logger->warning("Skipping project $projectFullName because of empty or invalid support level.");
```

Comment:
Better to use `sprintf("Skipping project %s because of empty or invalid support level.", $projectFullName)`?